### DISCUSSION TITLE
Channel Set import detects duplicate fields, but should be allowed to use those instead of asking to rename

### DISCUSSION THREAD

**Discussion by 5BYFIVE** [ExpressionEngine/ExpressionEngine#2982] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
A lot of channels I’m importing share fields across channels. This makes the import ask to rename each duplicated field and upload destination. Might be nice to be able to use what’s already there.

**Describe the solution you'd like**
Maybe there is a way to indicate "I don't want to duplicate, use the original detected field" with some kind of field.

**Describe alternatives you've considered**

**Teachability, Documentation, Adoption, Migration Strategy**

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by robsonsobral** [comment]
This causes a lot of issues when you have many channels to import, it doesn't matter whether they're into one or several files. We should at least be able to submit the IDs of channels, category groups, categories and fields.

**Comment by jcogs-design** [comment]
Being able to import / export any associated channel layout definitions at the same time would be a nice improvement to this feature.

**Comment by rmaclemale** [comment]
I would really, really like this to be fixed.  I'm going to be managing a lot of EE sites, and when I add a new channel to one, it'd be nice to easily add that channel to the others by export/import without having to make a lot of different fields.
